scene/mission-replay/src/main/resources/fxml/MissionView.fxml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.layout.*?>
+
+<?import com.esri.arcgisruntime.mapping.view.SceneView?>
+<?import javafx.scene.control.ComboBox?>
+<?import java.lang.String?>
+<?import com.esri.arcgisruntime.mapping.view.MapView?>
+<?import javafx.collections.*?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.Slider?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.ToggleButton?>
+<?import com.esri.samples.scene.CameraModel?>
+<?import com.esri.samples.scene.AnimationModel?>
+<?import com.esri.samples.scene.PlaneModel?>
+<StackPane fx:controller="com.esri.samples.scene.MissionController"
+           xmlns:fx="http://javafx.com/fxml">
+    <!--Instantiate model instances to bind to-->
+    <fx:define>
+        <CameraModel fx:id="cameraModel" distance="500.0" angle="75.0" follow="true">
+        </CameraModel>
+        <AnimationModel fx:id="animationModel" keyframe="0">
+        </AnimationModel>
+        <PlaneModel fx:id="planeModel">
+        </PlaneModel>
+    </fx:define>
+    <!--SDK SceneView-->
+    <SceneView fx:id="sceneView">
+    </SceneView>
+    <!--Mission Selector Pane-->
+    <VBox StackPane.alignment="TOP_LEFT" spacing="5" maxWidth="200" maxHeight="50" styleClass="pane">
+        <ComboBox fx:id="missionSelector" promptText="Select a Mission" onAction="#changeMission">
+            <items>
+                <FXCollections fx:factory="observableArrayList">
+                    <String fx:value="GrandCanyon.csv">
```

Comment:
For these values that just take up one line we could do `<String fx:value="GrandCanyon.csv"/>`